spec.html

```diff
@@ -24636,7 +24635,7 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Let _value_ be ? BindingClassDeclarationEvaluation of |ClassDeclaration|.
           1. Let _className_ be the sole element of BoundNames of |ClassDeclaration|.
-          1. If _className_ is *"\*default\*"*, then
+          1. If _className_ is ~default~, then
```

Comment:
I don't actually understand why we're making a binding here at all. Possibly some interaction with module instantiation I'm missing? Anyone know what's up with that?